### PR TITLE
Add metadata libosinfo checking for v2v

### PR DESCRIPTION
**Pull request by xiaodwan**
Signed-off-by: Xiaodai Wang <xiaodwan@redhat.com>

### CODE REVIEW

**Comment by xiaodwan** [comment]
(1/1) type_specific.io-github-autotest-libvirt.convert_vm_to_libvirt.esx.vm.6_7.linux.latest8.arch_x86_64.it_vddk: PASS (394.28 s)
(1/1) type_specific.io-github-autotest-libvirt.convert_vm_to_libvirt.esx.vm.6_7.linux.latest7.arch_x86_64.it_vddk: PASS (434.02 s)
(1/1) type_specific.io-github-autotest-libvirt.convert_vm_to_libvirt.esx.vm.6_7.windows.win2016.arch_x86_64.it_vddk: PASS (635.00 s)
 (1/2) type_specific.io-github-autotest-libvirt.function_test_esx.positive_test.suse.opensuse42_3.esx_67.libvirt.it_vddk.vpx_uri: PASS (398.97 s)                                            
 (2/2) type_specific.io-github-autotest-libvirt.function_test_esx.positive_test.suse.opensuse42_3.esx_67.rhev.it_vddk.vpx_uri: PASS (1669.26 s)

**Comment by xiaodwan** [comment]
@chloerh @chunfuwen Please help to review this pr, thanks~

**Comment by chloerh** [comment]
The rest is ok to me

File: provider/v2v_vmcheck_helper.py [hunk 1]
```diff
@@ -127,6 +129,58 @@ def get_expect_video_model(self):
...
+        # Skip checking if any of them is not in supported list
+        if self.params.get('target') not in supported_output or self.params.get(
+                'output_mode') not in supported_output:
+            reason = 'target or output_mode is not in %s' % supported_output
+            logging.info('Skip Checking metadata libosinfo parameters: %s' % reason)
+            return
+
+        cmd = 'osinfo-query os --fields=short-id | tail -n +3'
+        # Too much debug output if verbose is True
+        output = process.run(cmd, timeout=20, verbose=False, shell=True, ignore_status=True)
```
[the remark this answers is not in the document]
Comment (xiaodwan) [reply provider/v2v_vmcheck_helper.py]: Done, thanks~

**Pull request merged**